<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2" rowstretch="2,0,0,0,0,0,0,0">
         <item row="3" column="0">
          <widget class="QFrame" name="frame_3">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(246, 246, 246);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="1" column="1">
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <family>맑은 고딕</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>생년월일</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QFrame" name="frame_5">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(246, 246, 246);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>맑은 고딕</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>계좌번호</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0" colspan="5">
          <widget class="QFrame" name="frame_2">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 207, 0);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="font">
               <font>
                <family>맑은 고딕</family>
                <pointsize>11</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>KB 국민은행 빠른 조회 서비스</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="4" column="4">
          <widget class="QRadioButton" name="six_month">
           <property name="text">
            <string>6개월</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QRadioButton" name="three_month">
           <property name="text">
            <string>3개월</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QRadioButton" name="one_day">
           <property name="text">
            <string>당일</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0" colspan="5">
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="0" column="0">
             <widget class="QPushButton" name="inf_input">
              <property name="font">
               <font>
                <family>맑은 고딕</family>
                <pointsize>11</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 207, 0);</string>
              </property>
              <property name="text">
               <string>입력</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QFrame" name="frame_4">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(246, 246, 246);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="1">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <family>맑은 고딕</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>계좌 비밀번호</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QRadioButton" name="one_month">
           <property name="text">
            <string>1개월</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="4">
          <widget class="QFrame" name="frame_9">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_9">
            <item row="0" column="0">
             <widget class="QLineEdit" name="birth">
              <property name="font">
               <font>
                <family>맑은 고딕</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
              <property name="inputMask">
               <string/>
              </property>
              <property name="maxLength">
               <number>6</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="1" colspan="4">
          <widget class="QFrame" name="frame_8">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="0" column="0">
             <widget class="QLineEdit" name="acc_pw">
              <property name="font">
               <font>
                <family>맑은 고딕</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
              <property name="inputMask">
               <string/>
              </property>
              <property name="maxLength">
               <number>4</number>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1" colspan="4">
          <widget class="QFrame" name="frame_7">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="0">
             <widget class="QLineEdit" name="acc_num">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>맑은 고딕</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);</string>
              </property>
              <property name="inputMask">
               <string/>
              </property>
              <property name="maxLength">
               <number>14</number>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Normal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="5" column="1" colspan="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>시작일</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="3" colspan="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>종료일</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="1" colspan="2">
          <widget class="QDateEdit" name="start_date"/>
         </item>
         <item row="6" column="3" colspan="2">
          <widget class="QDateEdit" name="end_date"/>
         </item>
         <item row="4" column="0" rowspan="3">
          <widget class="QFrame" name="frame_6">
           <property name="font">
            <font>
             <family>맑은 고딕</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(246, 246, 246);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="1">
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <family>맑은 고딕</family>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>입력 일수</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>383</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>acc_num</tabstop>
  <tabstop>acc_pw</tabstop>
  <tabstop>birth</tabstop>
  <tabstop>one_day</tabstop>
  <tabstop>one_month</tabstop>
  <tabstop>three_month</tabstop>
  <tabstop>six_month</tabstop>
  <tabstop>start_date</tabstop>
  <tabstop>end_date</tabstop>
  <tabstop>inf_input</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
